### PR TITLE
[WIP] - added day3

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Hi,

I started day 3 documentation unless Layout system is actualy broken.
@crouillon is investigating to allow me create and use correct layout from BackBee "Editor".

You can start the review :)

### CODE REVIEW

**Comment by fkroockmann** [comment]
I dont understand what day3 is, explanation of content or layout creation or both ?

**Comment by fkroockmann** [comment]
I think the day3 not contain an explanation of the contents . It should no longer explain what a layout saying that there are editable areas etc., explain how to make a layout "final" , that is to say that one can not create sub pages, explain different data as " mainZone " ," defaultClasscontent " . It should also say that a layout is linked to a site.
The explanation of the contents in this step seems too just, here you explain a small part of content could make the user feel confused .

**Comment by mickaelandrieu** [comment]
thank you for review @fkroockmann, I will improve my work :)

**Comment by ndufreche** [comment]
For me is missing the code we to put in each layout file.

Creation of repository/Layout/Main.twig:
```
<!DOCTYPE html>

<html>
    <head lang="en">
        <meta charset="utf-8"></meta>
        <title>BlogBee - Home</title>
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
        <link rel="stylesheet" href="{{ this.getUri("css/main.css") }}">
    </head>

    <body>
        {{ this.bb5toolbar()|raw }}

        <div class="container-fluid">
            <div class="header-section">

                <header class="row header page-header">
                    {{ this.partial("partials/header.twig")|raw }}
                </header>

                <nav class="navbar navbar-default">
                    <div class="container-fluid">
                        <div class="collapse navbar-collapse row">

                            <section class="hidden-xs category-list-section col-sm-9">
                                {{ this.partial("partials/navigation.twig")|raw }}
                            </section>

                            <section class="col-xs-12 col-sm-3 search-section">
                                {{ this.partial("partials/search.twig")|raw }}
                            </section>

                        </div>
                    </div>
                </nav>
            </div>

            <div class="row">
                {% block content %}{% endblock %}
            </div>

            <footer class="footer page-footer">
                {{ this.partial("partials/footer.twig")|raw }}
            </footer>

        </div>
        <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
        <script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
    </body>
</html>
```

Creation of repository/Layout/OneColumn.twig:
```twig
{% extends "Main.twig" %}

{% block content %}
<div class="col-xs-12">
    {{ this.container().first()|raw }}
</div>
{% endblock %}
```

Creation of repository/Layout/TwoColumn.twig:
```twig
{% extends "Main.twig" %}

{% block content %}
<div class="col-sm-8 col-xs-12">
    {{ this.container().first()|raw }}
</div>
<div class="col-sm-4 col-md-3 col-md-offset-1 hidden-xs">
    <aside>
        {{ this.container().next()|raw }}
    </aside>
</div>
{% endblock %}
```

And modification of repository/Layout/Article.twig:
```twig
{% extends "TwoColumn.twig" %}
```

**Comment by ndufreche** [comment]
And the partial code

Creation of repository/Templates/scripts/patrials/header.twig:
```twig
<div class="col-xs-8">
    <h1 class="site-title h1"><a class="link-unstyled" href="index.html">BlogBee <small>community blog</small></a></h1>
</div>
<section class="login-section col-xs-4 text-righ">
    <button class="btn btn-success pull-right" href="/login" data-toggle="modal" data-target="#authentication-modal">login / create account</button>
</section>
```

Creation of repository/Templates/scripts/patrials/navigation.twig:
```twig
<ul class="nav navbar-nav list-unstyled list-inline row">
    <li class="col-sm-2 text-nowrap"><a href="category.html#1">category 1</a></li>
    <li class="col-sm-2 text-nowrap"><a href="category.html#2">category 2</a></li>
    <li class="col-sm-2 text-nowrap"><a href="category.html#3">category 3</a></li>
    <li class="col-sm-2 text-nowrap"><a href="category.html#4">category 4</a></li>
    <li class="col-sm-2 text-nowrap"><a href="category.html#5">category 5</a></li>
    <li class="col-sm-2 text-nowrap"><a href="category.html#6">category 6</a></li>
</ul>
```

Creation of repository/Templates/scripts/patrials/search.twig:
```twig
<form action="/search" method="post" class="form-inline">
    <div class="form-group">
        <div class="input-group">
            <span class="input-group-addon btn-search-form btn-reset"><button type="reset" class="btn btn-warning glyphicon glyphicon-remove-sign"></button></span>
            <input type="text" class="input form-control" placeholder="search">
            <span class="input-group-addon btn-search-form btn-submit"><button type="submit" class="btn btn-primary glyphicon glyphicon-search"></button></span>
        </div>
    </div>
</form>
```

Creation of repository/Templates/scripts/patrials/footer.twig:
```twig
<ul class="row list-unstyled">
    <li class="col-xs-3 text-left">
        <a href="http://www.backbee.com/">Build with BackBee</a>
    </li>
    <li class="col-xs-3 text-center">
        <a href="https://backbee.github.com/blogbee">BlogBee tuorial</a>
    </li>
    <li class="col-xs-3 text-center">
        <a href="http://github.com/backbee/blogbee">github repository</a>
    </li>
    <li class="col-xs-3 text-right">
        <a href="http://www.lp-digital.fr/en/">lp digital system &copy;</a>
    </li>
</ul>

```

File: doc/day3.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+Introduction
+---------------
+
+BackBee is a complete and flexible content management system. If architecture is a bit complex to understand, you will see that it is subsequently very easy to quickly build any type of website.
+
+
+What for today?
+-------------------
+
+Today we will learn how BackBee manages content. This will be a somewhat difficult day and we recommend you to reread this tutorial to understand the system. By late afternoon, you will understand how to integrate the templates that we built in the second day and have created our layouts for our pages.
```
Comment (fkroockmann) [inline doc/day3.md]: In this part, I think we should rather explain what you will do on layouts etc.

File: doc/day3.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+Let's start with a schema which represent the BackBee architecture of our *Article page*:
+
+![the BackBee big picture](http://i.imgur.com/sLLJ19x.png "the BackBee big picture")
+
+Each element have his object representation inside **BackBee core library**:
+
+ - **Site** (``BackBuilder\Site\Site``) is your application.
+ - **Page** (``BackBuilder\NestedNode\Page``) represent each page of your application, for example for blogbee have three pages: home page, article page and author page.
+ - **Layout** (``BackBuilder\Site\Layout``) represent the layout of a page.
+ - **ContentSet** (``BackBuilder\ClassContent\ContentSet``) represent the blocs/columns of your layouts, there are blocs can be editables by user.
```
Comment (fkroockmann) [inline doc/day3.md]: Add: see more on day 4

File: doc/day3.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+It's time to work!
+
+In our website we will have static and dynamic contents as you can see in this detailled mockup:
+
+![Layout decomposition](http://i.imgur.com/2KiVJxd.png "Layout decomposition")
+
+We need to create two layouts:
+
+ - A "main-column" layout for author related pages
```
Comment (fkroockmann) [inline doc/day3.md]: Change name to "one-column" instead of "main-column"

File: doc/day3.md [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+ - A "two-columns" layout, for home and article pages
+
+### Creation of layouts
+
+Connect to BackBee and select **Templates** tab on top of the editor.
```
Comment (fkroockmann) [inline doc/day3.md]: Simply repeat how to connect to backbee

File: doc/day3.md [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+
+BackBee embed a lot of common layouts  and the layouts we need are already provided by the CMS.
+
+ 1. Click on "Others templates", and select "Default template" which is only composed of one single column.
+
+ ![Author layout](http://i.imgur.com/LUQTsiv.png "Author layout")
+
+ 2. Edit the Template name and set it to *OneColumn* and to the same for a "two columns" layout that we will name "TwoLayouts".
+
+You will see two files generated inside ``repository/Layouts`` :  ``Author.twig`` and ``Article.twig``, where we will insert our previous Bootstrap 3 templates from **Day 2**, and the calls to BackBee.
```
Comment (fkroockmann) [inline doc/day3.md]: The file generated will be Two-columns.twig and One-column.twig (if you follow my advice).

File: doc/day3.md (hunk 2 above)
Comment (mickaelandrieu) [inline doc/day3.md]: ok

**Comment by mickaelandrieu** [comment]
Thank you @fkroockmann for your review ! I will improve my work.

File: doc/day3.md [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
+
+We have used [Twig](http://twig.sensiolabs.org/) as templating engine, in order to extends [template inheritance](http://twig.sensiolabs.org/doc/tags/extends.html "Twig extends tag"), but BackBee also supports old simple *phtml* templating engine.
+
+Then, add ``Main.twig`` file in the ``repository\Layout`` folder which will acts as a parent of each layout.
```
Comment (fkroockmann) [inline doc/day3.md]: I don't understand why you say to add Main.twig

File: doc/day3.md (hunk 1 above)
Comment (mickaelandrieu) [inline doc/day3.md]: Like ? Tell me what did you expect in this part.

**Pull request merged**